app/api/role_invites.py

```diff
@@ -168,7 +168,8 @@ def accept_invite():
     return jsonify({
         "email": user.email,
         "event": role_invite.event_id,
-        "name": user.fullname if user.fullname else None
+        "name": user.fullname if user.fullname else None,
+        "role": uer.role.name
```

Comment:
Fix the typo.

Reply:
@prateekj117 It's not a typo :laughing: , it's a `uer` object, `users_events_roles`

Reply:
@uds5501 Can you name more understandable,At first even i thought its a typo

Reply:
@kushthedude I think that wouldn't exactly be necessary. all through our code, `uer` is being used as `users_events_roles` model so I think I should stick to it to maintain uniformity.